resources/reproducing-ci.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: default
+nav_order: 3
+parent: Resources
+grand_parent: RAPIDS Maintainer Docs
+title: Reproducing CI
+---
+
+# {{ page.title }}
+{:.no_toc}
+
+## Overview
+{:.no_toc}
+
+This page outlines some helpful information about reproducing CI builds and tests locally.
+
+At this time, this information only applies to `conda` related builds and tests.
+
+### Intended audience
+{: .no_toc }
+
+Operations
+{: .label .label-purple}
+
+Developers
+{: .label .label-green}
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+## Reproducing CI Jobs
+
+The GitHub Action jobs that power RAPIDS' CI are simply a collection of shell scripts that are run inside of our [CI containers](https://github.com/rapidsai/ci-imgs).
+
+This makes it easy to reproduce build and test issues from CI on local machines.
+
+To get started, you should first identify the image that's being used in a particular GitHub Action job.
+
+This can be done by inspecting the _Intialize Containers_ step as seen in the screenshot below.
+
+![](/assets/images/reproducing-ci/container.png)
+
+After the container image has been identified, you can volume mount your local repository into the container with the command below:
+
+```sh
+docker run \
+  --rm \
+  -it \
+  --gpus all \
+  --network=host \
+  --volume $PWD:/repo \
+  --workdir /repo \
+  rapidsai/ci:cuda11.8.0-ubuntu22.04-py3.10
+```
+
+This command makes the following assumptions:
+
+- Your current directory is the repository that you wish you troubleshoot
+- Your current directory has the same commit checked out as the pull-request whose jobs you are trying to debug
+
+A few notes about the flags above:
+
+- Most of the RAPIDS conda builds occur on machines without GPUs. Only the tests require GPUs. Therefore, you can omit the `--gpus` flag when running local conda builds
+- The `--network` flag ensures that the container has access to the VPN connection on your host machine. VPN connectivity is required for test jobs since they need access to [downloads.rapids.ai](https://downloads.rapids.ai) for downloading build artifacts from a particular pull-request. This flag can be ommitted for build jobs
+
+After you run the command above, you will be able to run commands inside the container.
+
+From here you can run any of the CI scripts:
+
+```sh
+# to build cpp...
+./ci/build_cpp.sh
+
+# to test cpp...
+./ci/test_cpp.sh
+
+# to build python...
+./ci/build_python.sh
+
+# to test python...
+./ci/test_python.sh
+
+# to test notebooks...
+./ci/test_notebooks.sh
+
+# to build docs...
+./ci/build_docs.sh
+```
+
+## Additional Considerations
+
+There are a few additional considerations for running CI jobs locally.
+
+### GPU Driver Versions
+
+RAPIDS' CI test jobs may run on one of many GPU driver versions. When you're trying reproduce CI test failures locally, it's important to pay attention to both the driver version used in CI and the driver version on your local machine.
```

Comment:
I'm a weak `-1` on making RAPIDS possessive. I think it makes sense and sounds better without being possessive. e.g. Apple iPhone vs. Apple's iPhone.
```suggestion
RAPIDS CI test jobs may run on one of many GPU driver versions. When reproducing CI test failures locally, it's important to pay attention to both the driver version used in CI and the driver version on your local machine.
```